src/main/java/com/jcraft/jsch/JulLogger.java

```diff
@@ -1,12 +1,20 @@
 package com.jcraft.jsch;
 
 import java.util.logging.Level;
+import java.util.logging.Logger;
 
 public class JulLogger implements com.jcraft.jsch.Logger {
 
-  private static final java.util.logging.Logger logger = java.util.logging.Logger.getLogger(JSch.class.getName());
+  private static final java.util.logging.Logger stlogger = java.util.logging.Logger.getLogger(JSch.class.getName());
+  private final java.util.logging.Logger logger;
```

Comment:
I think since you've added an `import java.util.logging.Logger;` statement, there is no longer a need to specify the fully qualified package for these?

Reply:
Although I wonder if it would be better to drop the `import java.util.logging.Logger` statement instead and just always fully qualify the package everywhere? That would make it slightly easier to reason about which `Logger` class is being used.